### PR TITLE
Ux share structure

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#5178

*Description of changes:*

- UX updates to Share Structure

### CODE REVIEW

File: src/interfaces/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (severinbeauvais) [inline src/interfaces/package-lock.json:1]: That's weird -- how did this file get created? I don't see package-lock files for any other component in this repo.

Either all shared components should have a package-lock file, or none should (I think).

Comment (cameron-freshworks) [reply src/interfaces/package-lock.json:1]: ahh, perhaps during the hoist. Will clean these up

**Review by severinbeauvais — APPROVED** [review]
GTG when you're happy with it.

File: src/interfaces/package-lock.json (hunk 1 above)
Comment (cameron-freshworks) [reply src/interfaces/package-lock.json]: Done 👍

**Pull request merged**